platform/pensando/dpu/systemd/dpu.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
```

Comment:
@ashwin-h what is the role of this dpu service?

Reply:
This service is starting docker-dpu.